app/pages/certification-redirect.tsx

```diff
@@ -0,0 +1,123 @@
+import React, {Component} from 'react';
+import {Row, Col, Button, Form} from 'react-bootstrap';
+import {graphql} from 'react-relay';
+import {NextRouter} from 'next/router';
+import moment from 'moment';
+import {CiipPageComponentProps} from 'next-env';
+import {certificationRedirectQueryResponse} from 'certificationRedirectQuery.graphql';
+import DefaultLayout from '../layouts/default-layout';
+
+interface Props extends CiipPageComponentProps {
+  query: certificationRedirectQueryResponse['query'];
+  router: NextRouter;
+}
+
+class CertificationRedirect extends Component<Props> {
+  static query = graphql`
+    query certificationRedirectQuery($rowId: String!) {
+      query {
+        session {
+          ciipUserBySub {
+            id
+          }
+          ...defaultLayout_session
+        }
+        certificationUrlByRowId(rowId: $rowId) {
+          id
+          rowId
+          createdAt
+          applicationByApplicationId {
+            id
+            facilityByFacilityId {
+              facilityName
+              organisationByOrganisationId {
+                operatorName
+              }
+            }
+          }
+        }
+      }
+    }
+  `;
+
+  render() {
+    const {query, router} = this.props;
+    const {session} = query;
+    const facility =
+      query?.certificationUrlByRowId?.applicationByApplicationId
+        ?.facilityByFacilityId;
+    const facilityName = facility?.facilityName;
+    const organisationName =
+      facility?.organisationByOrganisationId?.operatorName;
+    const applicationId = this.props.router.query.id;
+    const redirectURI = `/certify?applicationId=${applicationId}`;
+
+    const createdAtMoment = query?.certificationUrlByRowId?.createdAt
+      ? moment(query?.certificationUrlByRowId?.createdAt)
+      : null;
+    const currentMoment = moment();
+    // TODO(Dylan): handle expiry of url validation on the back end
+    // Sets an expiry of 7 days for the certification URL
```

Comment:
If you're creating the mutation chain to certify in this PR, you could handle the expiry there. Another option is to use a computed column. Doing both would be best

Reply:
I added a trigger to create an expires_at column